src/__Plugin.ino

```diff
@@ -1200,14 +1200,46 @@ byte PluginCall(byte Function, struct EventStruct *event, String& str)
     case PLUGIN_WRITE:
     case PLUGIN_REQUEST:
     {
-      for (taskIndex_t task = 0; task < TASKS_MAX; task++)
+      taskIndex_t firstTask = 0;
+      taskIndex_t lastTask = TASKS_MAX;
+      String command = String(str);                         // Local copy to avoid warning in ExecuteCommand
+      int dotPos = command.indexOf(".");                    // Find first period
+      if (Function == PLUGIN_WRITE                          // Only applicable on PLUGIN_WRITE function
+        && dotPos > -1) {                                   // First precondition is just a quick check for a period (fail-fast strategy)
+        String thisTaskName = command.substring(1, dotPos); // Extract taskname
+        thisTaskName.replace("[", "");
+        thisTaskName.replace("]", "");
+        if (thisTaskName.length() > 0) {                    // Second precondition
+          taskIndex_t thisTask = findTaskIndexByName(thisTaskName);
```

Comment:
Maybe also nice to add an extra check, if the task name is not found.
For example I know some users keep using duplicate task names and they may want to use an integer for the task index (`1 ... TASK_MAX` instead of `0 ... TASK_MAX-1`)
So `if (!validTaskIndex(thisTask))` try to see if it is a valid numerical and then see if that can be translated into a valid task index.